README.md

```diff
@@ -217,6 +219,52 @@ The following properties are expected to be present on the deployment `context`
 - `commandLineArgs.revisionKey` (provided by [ember-cli-deploy][5])
 - `deployEnvironment`           (provided by [ember-cli-deploy][5])
 
+## What if my Redis server isn't publicly accessible?
```

Comment:
Awesome work @nikz . Could you please just make a couple of small changes for me?

1) Move this stuff above the `Prerequisites` section so that `Prerequisites` is last.
2) Put your two new sections H3's underneath an H2 section (I'll let you decide on the name). Make is like the sections above which were `## Activitation -> ### When does activation occur?`